testsettings.py

```diff
@@ -22,29 +29,27 @@
     'NOSE_DB_TEST_CONTEXT': 'corehq.tests.nose.HqdbContext',
     'NOSE_NON_DB_TEST_CONTEXT': 'corehq.tests.nose.ErrorOnDbAccessContext',
 
-    # ignore record_deploy_success.py because datadog may not be installed
-    # (only matters when running --with-doctests)
-    'NOSE_IGNORE_FILES': '^(localsettings|record_deploy_success\.py)',
+    'NOSE_IGNORE_FILES': '^localsettings',
+
+    'NOSE_EXCLUDE_TESTS': ';'.join([
+        # FIXME failing, excluded for now because they were not run by django test runner
+        'corehq.apps.ota.tests.digest_restore.DigestOtaRestoreTest',
```

Comment:
That's a very good question, and it makes me think that this isn't working. I think I'll remove it.